alacritty_terminal/src/grid/storage.rs

```diff
@@ -221,8 +237,9 @@ impl<T> Storage<T> {
         }
     }
 
+    // TODO: The a & b lines here seem to start from 0 at the top and grow downwards.
     pub fn swap_lines(&mut self, a: Line, b: Line) {
-        let offset = self.inner.len() + self.zero + *self.visible_lines;
+        let offset = self.zero + *self.visible_lines;
```

Comment:
@chrisduerr this seems like something that should be safe to remove since `x + y % x == y % x`.